server/src-rsr/migrations/41_to_42.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE hdb_catalog.event_triggers
+  DROP CONSTRAINT event_triggers_schema_name_table_name_fkey;
```

Comment:
This unfortunately breaks cascading effects when renaming tables. If you rename a table with event triggers, it throws inconsistent metadata and does not allow renaming.